src/frontend.jl

```diff
@@ -48,6 +48,13 @@ other environments.
 plotsize() = Atom.plotsize()
 
 """
+    showprofile()
+
+Show currently collected profile information in the editor.
+"""
+showprofile() = isactive() && Atom.Profiler.profiler()
```

Comment:
Because `profile` sounds like you want to profile something, not show a  already recorded profile. I'd totally be up to renaming though :)
`profiletree()` is a pretty good idea though.